internal/store/driver.go

```diff
@@ -298,6 +314,10 @@ func (d *Driver) calculateDomains() []ingressv1alpha1.Domain {
 					Domain: rule.Host,
 				},
 			}
+			if d.customMetadata != "" {
+				domain.Spec.NgrokAPICommon.Metadata = d.customMetadata
```

Comment:
I think here you should just be able to do `domain.Spec.Metadata = d.customMetadata`.

Reply:
updated these throughout